akka-stream/src/main/scala/akka/stream/impl/StreamLayout.scala

```diff
@@ -186,6 +186,9 @@ import scala.util.control.NonFatal
             case Inert ⇒ // nothing to be done
             case _     ⇒ ErrorPublisher(ex, "failed-VirtualProcessor").subscribe(s)
           }
+        case _ if t == null ⇒
+          // cancelled before onError(null), must throw NPE, rule 2:13
```

Comment:
`2.13` please

Reply:
I had `2:13` because I copied the comment from a previous line, but I've updated both now.